packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -133,7 +133,7 @@
         margin: -11px 0px 22px 0px; // first-level lists need padding
     }
 
-    .paragraph ul:not(.perseus-widget-radio) {
+    .paragraph ul:not(.perseus-widget-radio, .indicatorContainer) {
```

Comment:
**thought**: CSS whackamole continues. 😞

Reply:
We really need to do something about that.